### PR TITLE
[3.9] service catalog: create service and ssl certs for controller manager

### PR DESCRIPTION
**Pull request by jboyd01**
create k8s service and ssl certs for controller manager
change controller manager port & annotate it as secure
add Prometheus scrape for Catalog controller manager over ssl
change log levels for both API Server and Controller Manager to -v 3

### CODE REVIEW

File: roles/openshift_service_catalog/tasks/generate_certs.yml [hunk 1]
```diff
@@ -35,6 +45,15 @@
     signer_key: "{{ generated_certs_dir }}/ca.key"
     signer_serial: "{{ generated_certs_dir }}/apiserver.serial.txt"
 
+- name: Generating Controller Manager keys
+  oc_adm_ca_server_cert:
```
Comment (smarterclayton) [inline roles/openshift_service_catalog/tasks/generate_certs.yml]: Why not use service serving cert for these?

Comment (jboyd01) [reply roles/openshift_service_catalog/tasks/generate_certs.yml]: @smarterclayton are you talking about using the service annotation  service.alpha.openshift.io/serving-cert-secret-name  to create the key and cert for us?  That is how we did it upstream in service catalog, but here with Ansible we were already doing it this way for our api server so I just followed the pattern.

**Comment by jboyd01** [comment]
/hold

Should not be merged as https://github.com/openshift/origin/pull/19117  was not merged.  On hold until we determine if catalog will be revendored into 3.9.z.

**Pull request closed without merging**